packages/console/src/pages/EnterpriseSso/index.tsx

```diff
@@ -32,10 +36,20 @@ function EnterpriseSsoConnectors() {
   const { pathname } = useLocation();
   const { navigate } = useTenantPathname();
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { currentTenantId, isDevTenant } = useContext(TenantsContext);
+  const { data: currentPlan } = useSubscriptionPlan(currentTenantId);
   const [{ page }, updateSearchParameters] = useSearchParametersWatcher({
     page: 1,
   });
 
+  const isSsoEnabled = currentPlan?.quota.ssoEnabled;
+  const upgradePlan = useCallback(() => {
+    navigate(subscriptionPage);
+  }, [navigate]);
+  const onCreate = useCallback(() => {
+    navigate(createEnterpriseSsoPathname);
+  }, [navigate]);
```

Comment:
Nit: we can use a `handleCreation` function to wrap this logic instead of using 2 similar method.

Reply:
Updated.